src/Microsoft.Diagnostics.EventFlow.Outputs.Oms/OmsOutput.cs

```diff
@@ -3,6 +3,11 @@
 //  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
+using Microsoft.Diagnostics.EventFlow.Configuration;
```

Comment:
It does not really matter that much but we have a slight preference for keeping System namespaces above others. There is a setting in Visual Studio that helps maintain that order.

Reply:
@karolz-ms  Thanks. I'm using the "remove and sort" command in VS, and I like it - the sorting helps me locate using's faster.